app/controllers/graphql_devise/application_controller.rb

```diff
@@ -1,9 +1,10 @@
 # frozen_string_literal: true
 
 module GraphqlDevise
-  ApplicationController = if Rails::VERSION::MAJOR >= 5
-    Class.new(ActionController::API)
-  else
-    Class.new(ActionController::Base)
-  end
+  ApplicationController = Class.new(GraphqlDevise.base_controller)
```

Comment:
Would this fail in production with eager load? I think this would happen before the mount method gets executed during app start.